src/components/OpenSource/WhyOpenSource.js

```diff
@@ -0,0 +1,27 @@
+import React, { Fragment } from 'react'
+import SubtitleWithBody from '../Common/SubtitleWithBody'
+import styled from 'styled-components'
+
+const StyledSubtitleWithBody = styled(SubtitleWithBody)`
+  padding-top: ${props => props.theme.spacing['1']};
+  padding-bottom: ${props => props.theme.spacing['1']};
+  margin-top: ${props => props.theme.spacing['1']};
+  margin-bottom: ${props => props.theme.spacing['2']};
+`
+
+const StyledImage = styled.img`
```

Comment:
We want to use [our `Image` component](https://github.com/yldio/yld.io/blob/master/src/components/Common/Image.js) under the hood, rather than just `img` :pray: 

If you haven't seen it before, our `Image` component is used to ensure that we're playing nicely with the [`gatsby-image` plugin](https://www.gatsbyjs.org/docs/using-gatsby-image/) - a plugin which optimises images for us & helps with responsiveness, etc.

For example, this line in `Image.js`:
https://github.com/yldio/yld.io/blob/16e4e6b4034811823412e49d16adee9b7c23e353/src/components/Common/Image.js#L6

refers to the `fluid` prop that we get from the plugin. The fluid prop helps us with responsivity - [see the docs](https://www.gatsbyjs.org/packages/gatsby-image/#two-types-of-responsive-images).

You probably don't need to worry about understanding all of this right now. But at the very least, it's useful to know that all of this exists :smiley: